packages/@react-spectrum/s2/src/Badge.tsx

```diff
@@ -165,12 +165,12 @@ const badge = style<BadgeStyleProps>({
     fillStyle: {
       outline: {
         variant: {
-          accent: 'accent-800',
-          informative: 'informative-800',
-          neutral: 'gray-700',
-          positive: 'positive-700',
-          notice: 'notice-700',
-          negative: 'negative-800'
+          accent: lightDark('accent-800', 'accent-900'), // accent-visual-color
+          informative: lightDark('accent-800', 'accent-900'), // informative-visual-color
```

Comment:
mega nit: i think this should be `informative-800` and `informative-900` instead of `accent`